### PR TITLE
IAC blog update for 15.05.2020

### PR DESCRIPTION
**Pull request by daianamezdrea**

### CODE REVIEW

File: _posts/2020-05-14-status-update.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+  - litmus
+  - testing
+  - dsc
+  - pdk
+  - powershell
+  - windows
+  - modulebuilder
+  - voxpupuli
+---
+The IAC team has been working on a lot of interesting things over the past week. We want to share with our community some pretty interesting things. Here's a brief overview of what we've been up to:
```
Comment (DavidS) [inline _posts/2020-05-14-status-update.md:20]: "interesting things" duplicated - please reword

File: _posts/2020-05-14-status-update.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+## Community Contributions
+
+We think that is no doubt about our community contributors, they are always ready to help and they have excellent ideas. Here are some highlights of this week:
+
+Thank you to [akerl-unpriv](https://github.com/akerl-unpriv) for his contribution to the [puppetlabs-firewall](https://github.com/puppetlabs-firewall) module. We now have support for the `cgroup` flag in iptables.
+Thanks to [crazymind1337](https://github.com/crazymind1337) for [the great effort](https://github.com/puppetlabs/puppetlabs-docker/pull/607) cleaning up the `puppetlabs-docker` repo. It’s fantastic to see the open source community committed to keeping the standard high.
+
+##Paula continues her way back to Puppet goodies
+
+This is what she said when she was working on puppetlabs-apt release:
```
Comment (DavidS) [inline _posts/2020-05-14-status-update.md:47]: Please add a link to the release tooling here. Hopefully folx will want to see what paula is raving about here :-D

File: _posts/2020-05-14-status-update.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+  - module updates
+  - community
+  - litmus
+  - testing
+  - dsc
+  - pdk
+  - powershell
+  - windows
+  - modulebuilder
+  - voxpupuli
```
Comment (DavidS) [inline _posts/2020-05-14-status-update.md:18]: the tags need updating to reflect this week's contents.

File: _posts/2020-05-14-status-update.md [hunk 4]
```diff
@@ -0,0 +1,62 @@
+---
+layout: post
+title: "14st May 2020: IAC Team Status Update"
```
Comment (DavidS) [inline _posts/2020-05-14-status-update.md:3]: Today is the 15th.

File: _posts/2020-05-14-status-update.md [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+The IAC team has been working on a lot of interesting things over the past week. We want to share with our community some pretty interesting things. Here's a brief overview of what we've been up to:
+
+## COVID-19 who?
+We don't count anymore the days as a fully remote working team, but we count the days when we work hard and with all the energy we have for the best results!
+`#unstoppableteam #workHard #staysafe #thinkpositive`
+
+## Module Releases
+
+Modules released this week:
+- [puppetlabs/apt](https://github.com/puppetlabs/puppetlabs-apt) (7.4.2)
```
Comment (sheenaajay) [inline _posts/2020-05-14-status-update.md:29]: Ciaran and David released two more modules postgresql and mysql.

**Comment by DavidS** [comment]
https://github.com/puppetlabs/iac/pull/57 has been merged and could do with a mention of the docs new docs section "to be filled out over time with more information".

File: _posts/2020-05-14-status-update.md (hunk 5 above)
Comment (sanfrancrisko) [reply _posts/2020-05-14-status-update.md:29]: ...and firewall too

**Comment by DavidS** [comment]
rebased on top of the docs changes and fixed the link - apologies, that was my my fault for giving out the wrong one.

**Pull request merged**